CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 0.15.5
```

Comment:
is it possible to change version to `0.15.5-dev0`? I want to include a PR to fix issue in publishing `0.15.4` to PyPI in this release `0.15.5`. Details are in https://unstructuredai.slack.com/archives/C049XRFGR1R/p1723662226499499?thread_ts=1723658194.037349&cid=C049XRFGR1R

Reply:
Ahh okay cool! Updated to `-dev0`